examples/two-column-layout/style.css

```diff
@@ -59,9 +61,173 @@ body {
   transition: background 0.2s ease-out;
 }
 
+.aa-Item:hover .aa-ItemPicture img {
+  transform: scale(1.1);
+}
+
+.aa-ItemPicture {
+  width: 100%;
+  overflow: hidden;
+}
+
+.aa-ItemPicture img {
+  object-fit: cover;
+  width: 100%;
+  height: auto;
+  border-radius: 3px;
+  transition: transform 1.8s ease-out;
+  transform-origin: center bottom;
```

Comment:
I find the zoom-in transition from the bottom a bit weird, but it might just be me.